molecule/default/molecule.yml

```
@@ -37,7 +37,7 @@ provisioner:
           gitlab_runner_concurrent: 4
     host_vars:
       instancegitlabciopenstack1:
-        gitlab_runner_version: "16.5.0"
+        gitlab_runner_version: "16.9.0"
```

Should fix the pipeline

```suggestion
        gitlab_runner_version: "16.9.0-1"
```